views/p_s/metrics.sql

```diff
@@ -0,0 +1,116 @@
+-- | trx_rw_commits                                | 0                       ...| InnoDB Metrics - transaction         | NO      |
+-- | trx_undo_slots_cached                         | 0                       ...| InnoDB Metrics - transaction         | NO      |
+-- | trx_undo_slots_used                           | 0                       ...| InnoDB Metrics - transaction         | NO      |
+-- | memory_current_allocated                      | 138244216               ...| Performance Schema                   | PARTIAL |
+-- | memory_total_allocated                        | 138244216               ...| Performance Schema                   | PARTIAL |
+-- | NOW()                                         | 2015-05-31 13:27:50.382 ...| System Time                          | YES     |
+-- | UNIX_TIMESTAMP()                              | 1433042870.382          ...| System Time                          | YES     |
+-- +-----------------------------------------------+-------------------------...+--------------------------------------+---------+
+-- 412 rows in set (0.02 sec)
+
+CREATE OR REPLACE
+  ALGORITHM = TEMPTABLE
+  DEFINER = 'root'@'localhost'
+  SQL SECURITY INVOKER 
+VIEW metrics (
+  Variable_name,
+  Variable_value,
+  Type,
+  Enabled
+) AS
+(
+SELECT LOWER(VARIABLE_NAME) AS Variable_name, VARIABLE_VALUE AS Variable_value, 'Global Status' AS Type, 'YES' AS Enabled
+  FROM performance_schema.global_status
+) UNION ALL (
+SELECT NAME AS Variable_name, COUNT AS Variable_value,
+       CONCAT('InnoDB Metrics - ', SUBSYSTEM) AS Type,
+       IF(STATUS = 'enabled', 'YES', 'NO') AS Enabled
+  FROM information_schema.INNODB_METRICS
+  -- Deduplication - some variables exists both in GLOBAL_STATUS and INNODB_METRICS
+  -- Keep the one from GLOBAL_STATUS as it is always enabled and it's more likely to be used for existing tools.
+ WHERE NAME NOT IN (
+     'lock_row_lock_time', 'lock_row_lock_time_avg', 'lock_row_lock_time_max', 'lock_row_lock_waits',
+     'buffer_pool_reads', 'buffer_pool_read_requests', 'buffer_pool_write_requests', 'buffer_pool_wait_free',
+     'buffer_pool_read_ahead', 'buffer_pool_read_ahead_evicted', 'buffer_pool_pages_total', 'buffer_pool_pages_misc',
+     'buffer_pool_pages_data', 'buffer_pool_bytes_data', 'buffer_pool_pages_dirty', 'buffer_pool_bytes_dirty',
+     'buffer_pool_pages_free', 'buffer_pages_created', 'buffer_pages_written', 'buffer_pages_read',
+     'buffer_data_reads', 'buffer_data_written', 'file_num_open_files',
+     'os_log_bytes_written', 'os_log_fsyncs', 'os_log_pending_fsyncs', 'os_log_pending_writes',
+     'log_waits', 'log_write_requests', 'log_writes', 'innodb_dblwr_writes', 'innodb_dblwr_pages_written', 'innodb_page_size')
+) /*!50702
+  -- memory instrumentation available in 5.7.2 and later
+  UNION ALL (
+SELECT 'memory_current_allocated' AS Variable_name, SUM(CURRENT_NUMBER_OF_BYTES_USED) AS Variable_value, 'Performance Schema' AS Type,
```

Comment:
Prefix these variable names with "performance_schema_" please (to make them consistent with the other performance schema related variables).